config/database.php

```diff
@@ -114,6 +114,19 @@
             'schema' => 'public',
             'sslmode' => 'prefer',
         ],
+        'vectors' => [
+            'driver' => 'pgsql',
+            'host' => env('VECTORS_DB_HOST', env('DB_HOST', '127.0.0.1')),
+            'port' => env('VECTORS_DB_PORT', env('DB_PORT', '5432')),
+            'database' => env('VECTORS_DB_DATABASE', env('DB_DATABASE', 'forge')),
+            'username' => env('VECTORS_DB_USERNAME', env('DB_USERNAME', 'forge')),
+            'password' => env('VECTORS_DB_PASSWORD', env('DB_PASSWORD', '')),
+            'charset' => 'utf8',
+            'prefix' => env('VECTORS_PREFIX', ''),
```

Comment:
Since the vectors db holds references to models/ids in the main db, does that indicate that maybe this vector data belongs in the same db as all the referenced models? If so, we could make the `vectors` connection prefix all the vector tables (`vector_'), instead of connecting to a separate db entirely.